nessie/sql_templates/create_intermediate_schema.template.sql

```diff
@@ -65,26 +111,20 @@ AS (
         s.name AS canvas_section_name,
         et.name AS canvas_course_term,
         extracted_section_ids.sis_term_id,
-        extracted_section_ids.sis_section_id,
-        sc.course_display_name AS sis_course_name,
-        sc.course_title AS sis_course_title,
-        sc.instruction_format AS sis_instruction_format,
-        sc.section_num AS sis_section_num,
-        sc.is_primary AS sis_primary
+        extracted_section_ids.sis_section_id
```

Comment:
@raydavis Sorry for the drive-by - I think the bug you found in NS-46 isn't that Canvas-less SIS sections are missing, but that Canvas-less SIS sections have null `sis_term_id` and `sis_section_id`. Nessie SQL is pulling from the wrong side of the join and should (I think) be using `sc`, not `extracted_section_ids`. This is just a hunch; apologies if you're way ahead of me.